src/components/NumberInput/NumberInput.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { NumberInput } from './NumberInput';
+
+describe('Number input', () => {
```

Comment:
You should add checks for the dynamic attribute values set by props - `label`, `name`, `value`